webpack/scenes/ContentViews/Details/Versions/Compare/CVVersionCompareConfig.js

```diff
@@ -0,0 +1,188 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { TimesIcon, CheckIcon } from '@patternfly/react-icons';
+import {
+  selectPackageGroupsComparison,
+  selectPackageGroupsComparisonStatus,
+  selectRPMPackagesComparison,
+  selectRPMPackagesComparisonStatus,
+  selectFilesComparison,
+  selectFilesComparisonStatus,
+  selectErrataComparison,
+  selectErrataComparisonStatus,
+  selectModuleStreamsComparison,
+  selectModuleStreamsComparisonStatus,
+  selectDebPackagesComparison,
+  selectDebPackagesComparisonStatus,
+  selectDockerTagsComparison,
+  selectDockerTagsComparisonStatus,
+} from '../../ContentViewDetailSelectors';
+import {
+  getPackageGroupsComparison,
+  getRPMPackagesComparison,
+  getFilesComparison,
+  getErrataComparison,
+  getModuleStreamsComparison,
+  getDebPackagesComparison,
+  getDockerTagsComparison,
+} from '../../ContentViewDetailActions';
+
+export const TableType = PropTypes.shape({
+  name: PropTypes.string,
+  getCountKey: PropTypes.func,
+  responseSelector: PropTypes.func,
+  statusSelector: PropTypes.func,
+  autocompleteEndpoint: PropTypes.string,
+  fetchItems: PropTypes.func,
+  columnHeaders:
+    PropTypes.arrayOf(PropTypes.shape({
+      title: PropTypes.string,
+      width: PropTypes.number,
+      getProperty: PropTypes.func,
+    })),
+});
+
+export default ({
+  versionOne,
+  versionTwo,
+  versionOneId,
+  versionTwoId,
+}) => {
+  const compareContent = (item, versionId) => {
+    const {
+      comparison,
+    } = item;
+    if (
+      Number(comparison?.[0]) === Number(versionId)
+      || Number(comparison?.[1]) === Number(versionId)) {
+      return (
+        <CheckIcon />
+      );
+    }
+    return (
+      <TimesIcon />
+    );
+  };
+  return ([
+    {
+      name: __('RPM Packages'),
+      getCountKey: (itemVersionOne, itemVersionTwo) =>
+        itemVersionOne?.rpm_count || itemVersionTwo?.rpm_count,
+      responseSelector: state => selectRPMPackagesComparison(state, versionOneId, versionTwoId),
+      statusSelector: state => selectRPMPackagesComparisonStatus(state, versionOneId, versionTwoId),
+      autocompleteEndpoint: '/packages/auto_complete_search',
+      fetchItems: params => getRPMPackagesComparison(
+        versionOneId,
+        versionTwoId,
+        params,
+      ),
+      columnHeaders: [
+        {
+          title: __('Name'),
+          getProperty: item => item?.nvrea,
```

Comment:
Make this a URL just like version details config. We can do the same for content types that have a details view. The version details config should be a good pointer around which names should be a URL.
getProperty: item => (
          <a href={urlBuilder(`packages/${item?.id}`, '')}>
            {item?.nvrea}
          </a>),